src/www/pages/login.html

```diff
@@ -1,40 +1,42 @@
-
-
 <div ng-controller="controllerNavbar">
+    <div ng-if="!user.loggedIn">
+        <div class="row">
+            <div class="col-md-3 col-sm-1"></div>
+            <div class="col-md-6 col-sm-10 text-center login-form">
+                <form>
+                    <h1>Login</h1>
+                    <div class="form-group">
+                        <input type="text" ng-model="email" class="form-control login-input"
+                               placeholder="Email address">
+                    </div>
 
-  <div ng-if="!user.loggedIn">
-    <div class="row">
-      <div class="col-md-3 col-sm-1"></div>
-      <div class="col-md-6 col-sm-10 text-center login-form">
-        <form>
-          <h1>Login</h1>
-          <div class="form-group">
-            <input type="text" ng-model="email" class="form-control login-input" placeholder="Email address">
-          </div>
+                    <div class="form-group ">
+                        <input type="password" ng-model="password" class="form-control login-input"
+                               placeholder="Password">
+                    </div>
 
-          <div class="form-group ">
-            <input type="password" ng-model="password" class="form-control login-input" placeholder="Password">
-          </div>
-
-          <button type="submit" ng-click="login(email, password)" class="btn btn-primary btn-lg login-btn">
-            Login
-          </button>
-        </form>
-      </div>
-      <div class="col-md-3 col-sm-1"></div>
+                    <button type="submit" ng-click="login(email, password)" class="btn btn-primary btn-lg login-btn">
+                        Login
+                    </button>
+                </form>
+                <p style="margin-top: 1rem">If you do not have an account yet, you can get one via the weekly newsletter. <b>In the weekly
+                    newsletter is a registration link</b>.</p>
```

Comment:
"If you do not have an account yet, sign up via the registration link in the Weekly Newsletters" sounds better in my opinion